tests/Common/WebFrameworkTestCase.php

```diff
@@ -50,10 +50,18 @@ protected static function getEnvs()
     {
         $envs = [
             'DD_TEST_INTEGRATION' => 'true',
-            'DD_TRACE_ENCODER' => 'json',
+            'DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED' => 'true',
+
+            // Settings for the background sender (BGS)
+            'DD_TRACE_BGS_ENABLED' => true,
```

Comment:
I wonder if we should only include settings that are required, so we test a scenario as close as possible to our users' OOTB experience.

Here I would leave: `DD_TRACE_AGENT_FLUSH_AFTER_N_REQUESTS` and `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED` only (with the latter to be removed with next PR into master :) )

Reply:
I think we'll want to decrease the flush interval drastically or else the test suite will take forever to run -- since is fully async we basically have to wait for 2 interval cycles. So maybe defaults for everything except:
- `DD_TRACE_AGENT_FLUSH_AFTER_N_REQUESTS`
- `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED`
- `DD_TRACE_AGENT_FLUSH_INTERVAL`
What do you think? /cc @SammyK